inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -1319,6 +1319,69 @@ void GNAGraphCompiler::EltwisePrimitive(InferenceEngine::CNNLayerPtr layer) {
     }
 }
 
+void GNAGraphCompiler::GemmPrimitive(InferenceEngine::CNNLayerPtr layer) {
+    auto quantized = InferenceEngine::getInjectedData<QuantizedLayerParams>(layer);
+
+    IE_ASSERT(!layer->insData.empty());
+    IE_ASSERT(!layer->outData.empty());
+    IE_ASSERT(layer->insData.size() == 2);
+    auto input_1 = layer->insData[0].lock();
+    auto input_2 = layer->insData[1].lock();
```

Comment:
Please, add some comments explaining which input corresponds to weights and which corresponds to dynamic input and that they are passed in GNA in the opposite order. Or give corresponding names to variables.